packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -523,6 +522,26 @@ function getLargestRawDataEntryFromEntity(
   return largestItem;
 }
 
+/**
+ * Helper function to generate a map of property sizes for a given Entity.
+ * This is used to determine what property keys on an Entity have a particularly large value.
+ * Intended to be used when logging upload errors due to large payload size.
+ *
+ * @param data
+ * @returns
+ */
+function getPropSizeMapFromEntity(data: Entity): any {
+  const propSizeMap = {};
+
+  for (const [key, value] of Object.entries({
+    ...data,
```

Comment:
is deconstructing `data` necessary?

Reply:
not at all, holdover from when I was putting other stuff in here beyond data. Great catch!